backEndDeps.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This file should not exist. Each individual in-browser module should provide its own list of backend dependencies. See the jira issue for more info on that.